plugins/manipulator.cpp

```diff
@@ -2164,14 +3807,12 @@ void viewscreen_unitlaborsst::render()
     OutputString(10, x, y, Screen::getKeyDisplay(interface_key::CUSTOM_P));
     OutputString(15, x, y, ": Apply Profession ");
     OutputString(10, x, y, Screen::getKeyDisplay(interface_key::CUSTOM_SHIFT_P));
-    OutputString(15, x, y, ": Save Prof. ");
+    OutputString(15, x, y, ": Read Prof. ");
```

Comment:
I haven't looked around a lot, but I don't see anything dealing with the CUSTOM_SHIFT_P key that you changed, so I don't think changing this text is accurate. Let me know if I'm wrong.